### PR TITLE
Light Card Brightness Preset Buttons

### PR DESCRIPTION
**Pull request by benbur98**
Light Updates:

New Brightness Buttons to toggle 3 Brightness Presets
image
CSS Grid Code Updated slightly on Both Cards to be more flexible for future additions of rows

 The code change is tested and works locally.
 There is no commented out code in this PR.
 I have followed the [contribution guidelines](https://github.com/UI-Lovelace-Minimalist/UI/blob/main/.github/CONTRIBUTING.md)
Please feel free to test these changes out :)

- This PR fixes or closes issue: N/A - Enhancement
- This PR is related to issue: N/A
- Link to documentation pull request: N/A

## Checklist
- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [X] I have followed the [contribution guidelines](https://github.com/UI-Lovelace-Minimalist/UI/blob/main/.github/CONTRIBUTING.md)
    - [ ] This PR is for a custom-card or documentation change and therefore directed to the `main` branch.
    - [X] This PR is for a official card or any other directly to the integration related change and therefore directed to the `release` branch.

### CODE REVIEW

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml [hunk 1]
```diff
@@ -8,74 +8,91 @@ card_light:
...
               [[[
                   var color = entity.attributes.rgb_color;
-                  if (variables.ulm_card_light_enable_color == true){
-                    if(variables.ulm_card_light_force_background_color == true || hass.themes.darkMode){
-                      if (color){
-                        return 'rgba(' + color + ',0.1)'
+                  if (variables.ulm_card_light_enable_color) {
+                    if (variables.ulm_card_light_force_background_color || hass.themes.darkMode) {
+                      if (color) {
+                        return 'rgba(' + color + ',var(--opacity-bg))';
```
Comment (basbruss) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml:38]: Breaks the card when `ulm_card_light_force_background_color == true` and using light-mode.
`var(--opacity-bg)` = `1` in light moder
image

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml [hunk 2]
```diff
@@ -260,3 +286,34 @@ card_light:
             border-radius: 14px;
             box-shadow: none;
           }
+    item3:
+      card:
+        type: "horizontal-stack"
```
Comment (basbruss) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml:291]: Breaks the dashboard if `horizontal-stack` is not used somewhere else in the dashboard.
Fix is to use the internal templates for 3 items on a row:
`list_3_items`

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml [hunk 3]
```diff
@@ -260,3 +286,34 @@ card_light:
...
+        cards:
+          - type: "custom:button-card"
+            template: "widget_icon"
+            tap_action:
+              action: "call-service"
+              service: "light.turn_on"
+              service_data:
+                entity_id: "[[[ return entity.entity_id ]]]"
+                brightness: "[[[ return variables.ulm_card_light_brightness_low ]]]"
```
Comment (basbruss) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml:300]: Better to use `brightness_pct:` to improve UX.

**Review by basbruss — CHANGES_REQUESTED** [review]

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml (hunk 1 above)
Comment (benbur98) [reply custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml]: Have updated this. Having an issue now with the Colour around the 3 item list - will look into this.
image

Comment (basbruss) [reply custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml]: Just add a style to the list template to remove the background
```yaml
styles:
  card:
    - background: "none"
```

Comment (benbur98) [reply custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml]: Perfect, thanks for the tip.

**Pull request merged**